test/TestExpectations.json

```diff
@@ -3439,6 +3509,12 @@
     "expectations": ["SKIP"],
     "comment": "For some reason service_workers do not close in chrome-headless-shell"
   },
+  {
+    "testIdPattern": "[touchscreen.spec] Touchscreen Touchscreen.prototype.touchMove should throw if no touch was started",
+    "platforms": ["linux"],
```

Comment:
```suggestion
    "platforms": ["darwin", "linux", "win32"],
```

I assume it applies to all platforms

Reply:
actually only that one it seems, judging from the CI run

Reply:
CI only runs on Linux by default. I can add the full ci label but it would take longer to run.

Reply:
It seems like full ci passed, right? Is this change needed?

Reply:
interesting, that is only passed on linux but if that is the case then this change is not needed.